.github/workflows/main.yml

```diff
@@ -32,63 +29,22 @@ jobs:
           zlib1g-dev \
           strace \
           libssl-dev \
-          pkg-config
-
-    - name: Install rust
-      run: |
-        export RUSTUP_HOME="$HOME/.rustup"
-        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-        source "$HOME/.cargo/env"
-        cargo --version
-
-    - name: Update postgrestd git submodules
-      run: git submodule update --init --recursive
-
-    - name: Build postgrestd
-      run: ./run install
-
-  test_aarch64:
-    name: aarch64 build & test
-    runs-on: [self-hosted, linux, ARM64, postgrestd]
-
-    env:
-      STD_TARGETS: aarch64-postgres-linux-gnu
-
-    steps:
-    - uses: actions/checkout@v3
-
-    - name: Show kernel info
-      run: uname -a
-
-    - name: Print env
-      run: env
-
-    - name: Install system dependencies
-      run: |
-        sudo apt-get install -y \
-          clang-11 \
-          llvm-11 \
-          clang \
-          gcc \
-          make \
-          build-essential \
-          libz-dev \
-          zlib1g-dev \
-          strace \
-          libssl-dev \
-          pkg-config
+          pkg-config \
+          crossbuild-essential-arm64
 
     - name: Install rust
       run: |
         export RUSTUP_HOME="$HOME/.rustup"
         curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
         source "$HOME/.cargo/env"
         cargo --version
-        echo "RUSTUP_HOME=$HOME/.rustup" >> $GITHUB_ENV
-        echo "PATH=$HOME/.cargo/bin:$PATH" >> $GITHUB_ENV
```

Comment:
I am honestly surprised removing these two lines allowed for the build/install below to work, since in the past the runner environments had a heck of a time trying to find rustc if the environment variables were not persisted in `$GITHUB_ENV`

Reply:
I didn't remove those lines, they're not present in https://github.com/tcdi/postgrestd/blob/873dfbd32fd4df10916956f5a53c91c374fb8c77/.github/workflows/main.yml#L37 either

Reply:
The x86_64 runners from github almost certainly already have these on the path / in the env, so that's why it's not needed.